### DISCUSSION TITLE
suggestion for minor clarification

### DISCUSSION THREAD

**Discussion by philipschwarz** [tpolecat/doobie#1776] [Ideas]
Hello, FWIW, in the Diving Deeper section of https://github.com/tpolecat/doobie/blob/main/modules/docs/src/main/mdoc/docs/04-Selecting.md, it might be worth explaining what the `hi` stands for in `doobie.hi.connection` (or have I just missed such an explanation elsewhere?) and what the `H` stands for in `HC` (it looks like the `C` stands for connection?).

**Comment by jatcwang** [comment]
Hi @philipschwarz you're right I totally agree that the lack of docs around what these acronym means is very confusing. I don't believe there are is anything explaining what e.g. HC means. Contributions around this area will be welcome! Created #1777

**Comment by jatcwang** [comment]
I have updated the docs in https://tpolecat.github.io/doobie/docs/02-Toolkit.html with example modules like doobie.hi.HPS and explained what H/F stands for. Should appear next time the docs are updated :)
